pkg/ingester/transfer.go

```diff
@@ -279,7 +279,7 @@ func (i *Ingester) TransferTSDB(stream client.Ingester_TransferTSDBServer) error
 			if !ok {
 				file, err = createfile(f)
 				if err != nil {
-					return err
+					return errors.Wrap(err, fmt.Sprintf("unable to create the file %s to store incoming TSDB block", f))
```

Comment:
You can use `errors.Wrapf`, also I would remove 'the', as we name the file now.

Reply:
Done, thanks!